python/lsst/cp/pipe/astierCovPtcFit.py

```diff
@@ -70,10 +70,11 @@ def makeCovArray(inputTuple, maxRangeFromTuple=8):
     Parameters
     ----------
     inputTuple: `numpy.recarray`
-        Recarray with rows with at least (mu, cov, var, i, j, npix), where:
+        Recarray with rows with at least (mu, afwVar, cov, var, i, j, npix), where:
```

Comment:
I will explore the use of a NumPy structured array instead, as you suggest.  This part of the code was directly ported from an external repo by another researcher and, initially, I did not want to make too many structural changes so that it could work.